lib/glific/providers/gupshup/template.ex

```diff
@@ -70,12 +69,16 @@ defmodule Glific.Providers.Gupshup.Template do
       |> Templates.do_create_session_template()
       |> tap(fn _resp -> PartnerAPI.delete_local_resource(body[:media_url], attrs.shortcode) end)
     else
+
       {:error, error} ->
         Logger.error(error)
-        {:error, ["BSP", "couldn't submit for approval"]}
+        {:error, ["BSP", error]}
```

Comment:
Can we keep the error "Couldn't submit approval" and then add a secondary error which we get from subsequent functions